### DISCUSSION TITLE
Welcome DietPi Requests !

### DISCUSSION THREAD

**Discussion by MichaIng** [MichaIng/DietPi#4889] [Announcements]
## 👋 Welcome DietPi Requests !

We’re using GitHub Discussions as a place to connect with other members of our community, and build together a better DietPi platform:
* Image request
   Suggest a new SBC to be added to the official support and image list of DietPi
* Software request
   Suggest a software title to be added to DietPi-Software

Please continue to report **Feature requests** and **Bugs**, as **Issues**

**Comment by arium76** [comment]
Update Software request -  is there update coming out for ReadyMedia 1.3.3 ?  is there any easy way to update on DietPi instead of compiling source

**Reply by Joulinar** [reply]
We do not maintain minidlna (ReadyMedia) ourselves. It is installed from the Debian global package repository. https://packages.debian.org/search?suite=all&searchon=names&keywords=minidlna

Other versions than those available on the package server have to be compiled manually.

**Reply by MichaIng** [reply]
Debian Bullseye and Bookworm ship v1.3.0 and this won't change, only security patches and bug fixes will be applied/backported on demand. Debian Trixie (current testing version) ships v1.3.2 and will be most likely bumped to v1.3.3 shortly after its release. So if you are fine with living on bleeding edge and by times breaking changes with APT package upgrades, you could upgrade to Debian Trixie. Is there any particular change coming with ReadyMedia v1.3.3 you are waiting for?

**Comment by arium76** [comment]
thanks for the responses!

**Comment by hecatae** [comment]
could blocky be considered as an alternative to pihole and adguardhome?
https://github.com/0xERR0R/blocky

**Reply by MichaIng** [reply]
A Pi-hole and AGH alternative? Interesting. Could you make a dedicated software request for this?

**Reply by Joulinar** [reply]
done

**Comment by Cs137** [comment]
Would it be possible to add a 'Show and Tell' section, or something similar? I created this DietPi inspired [Homer theme](https://codeberg.org/Cs137/homer-theme-dietpi), that I would like to share with others.

**Reply by MichaIng** [reply]
We will think about it. At least we should add it to our docs 🙂: https://dietpi.com/docs/software/system_stats/#homer

**Reply by Cs137** [reply]
I could have thought of that myself - thanks for the tip! I can take some inspiration from the themes mentioned already and prepare a pull request. At worst, I should find time to do it at the weekend.

**Reply by Cs137** [reply]
It was very straight forward - thanks! [PR](https://github.com/MichaIng/DietPi-Docs/pull/1091)
